pennylane/operation.py

```diff
@@ -1477,6 +1492,8 @@ class Observable(Operator):
             can be useful for some applications where the instance has to be identified
     """
 
+    _queue_category = None
```

Comment:
Why is this `None`? Are we just initializing it here so that it can be overwritten by `MeasurementProcess` or `MidMeasure` depending on where the observable is called?

Reply:
Observables that don't also inherit from `Operation`, like `Tensor` or `Hamiltonian` , are currently not processed into `_prep`, `_ops`, or `_measurements`. 

`None` denotes "dont stick me in any queue".

Reply:
Do we have any like that ?

Reply:
Is this because they are encapsulated by a measurement process that is put into `_measurements`?